Mathlib/Analysis/NormedSpace/lpSpace.lean

```diff
@@ -885,13 +885,13 @@ theorem _root_.Memℓp.infty_pow {f : ∀ i, B i} (hf : Memℓp f ∞) (n : ℕ)
   (lpInftySubring B).pow_mem hf n
 #align mem_ℓp.infty_pow Memℓp.infty_pow
 
-theorem _root_.nat_cast_memℓp_infty (n : ℕ) : Memℓp (n : ∀ i, B i) ∞ :=
+theorem _root_.natCast_memℓp_infty (n : ℕ) : Memℓp (n : ∀ i, B i) ∞ :=
```

Comment:
This rename seems out of scope for this PR; the `nat_cast` vs `natCast` change (pending poll result) was the other one, right? Same for the `int_cast` below.